Chirp/src/Chirp.Infrastructure/AuthorRepository.cs

```diff
@@ -29,7 +24,7 @@ public async Task<AuthorDTO> GetAuthorByEmail(string email)
         
         var authors = await query.ToListAsync();
         
-        return authors.Select(author => new AuthorDTO(author.Name, author.Email)).FirstOrDefault();
+        return authors.Select(author => new AuthorDTO(author.Name, author.Email)).First();
```

Comment:
I think it should be FirstOrDefault, that way it would return null, if there is no match.
We could then implement that it would throw a "UserDosentExistException" or something else, if it was null, at some point.